scripts/server

```diff
@@ -21,6 +21,10 @@ then
     then
         docker-compose -f "${DIR}/../docker-compose.yml" \
                        -f "${DIR}/../docker-compose.airflow.yml" up
+    elif [ "${1:-}" = "--with-swagger" ]
```

Comment:
Will we ever want to run this command with `--with-airflow` and `--with-swagger`? I was unable to start both services from this script.